### PR TITLE
PROGRAMMES-6463: Fix empty side panel 

### PR DESCRIPTION
**Pull request by BjornTwachtmann**
Move fauxcolumn into a separate class, rather than having everything in map__column

### CODE REVIEW

**Comment by programmes-developers** [comment]
Can one of the admins verify this patch?

File: src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss [hunk 1]
```diff
@@ -32,6 +32,10 @@
         margin-bottom: -50em;
         padding-bottom: 50em;
     }
+    .map__fauxcolumn {
+        margin-bottom: -50em;
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss]: This is duplicated above.

Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss]: Fixed

File: src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss [hunk 2]
```diff
@@ -28,7 +28,8 @@
 }
 
 @include mq-range('bpw2') {
-    .map--episode .map__column .map__inner {
+    .map--episode .map__column .map__inner,
```
Comment (votemike) [inline src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss]: Is this declaration still needed for map__inner?

Comment (BjornTwachtmann) [reply src/Ds2013/Presenters/Section/Episode/Map/_episode-map.scss]: yeah, it's on more than one thing in V2 also, christ knows why, but we've gone far enough down this rabbit hole

**Pull request merged**